lib/elixir_make/artefact.ex

```diff
@@ -179,31 +190,15 @@ defmodule ElixirMake.Artefact do
       config[:make_precompiler_nif_versions] ||
         [versions: [current_nif_version]]
 
-    versions = nif_versions[:versions]
-
     Enum.reduce(targets, [], fn target, archives ->
-      archive_filenames =
-        Enum.reduce(versions, [], fn nif_version_for_target, acc ->
-          availability = nif_versions[:availability]
-
-          available? =
-            if is_function(availability, 2) do
-              availability.(target, nif_version_for_target)
```

Comment:
I don't think we can remove it. Probably we want to keep the code as it was and add this inside this branch:

    IO.warn(":availability key in elixir_make is deprecated, pass a function as :versions instead")

Reply:
Okay. I added the warning message.